packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -317,6 +320,13 @@ const CommonDomainRoutes = ({
   // GOVERNANCE END
 
   // DISCUSSIONS
+  <Route
+    key="/:scope/community-home"
```

Comment:
is this product decision that we will access the page with this URL like `dydx/community-home`? For me it does not look good. for me home page should be just `/dydx` or `dydx/dashboard` or something. Please consult it with product